crates/core/src/host/mod.rs

```diff
@@ -166,3 +167,12 @@ impl EnergyMonitor for NullEnergyMonitor {
     ) {
     }
 }
+
+#[derive(Debug, Display)]
+pub enum SysCall {
```

Comment:
I'd love to reconcile this with https://github.com/clockworklabs/SpacetimeDB/blob/c566dbae3dc759b3f712cd0fc4a5f172d32306ed/crates/core/src/host/wasm_common/instrumentation.rs#L84